File.downloadがエラーとなります

## 不具合の概要(Description of bug)
- SDKのバージョン(SDK version):v3.1.0
- 不具合が発生した環境(Environment requirement): Cordova+Android, iOS
- 不具合を再現させる手順・サンプルコード(Step for Reproduce bug / Sample code)
```
new NCMB(appKey, clientKey).File.download("sample.txt")
            .then(function(data) {
                console.log("File.download success data: " + data);
            })
            .catch(function(err) {
                console.log("error : " + err.message);
                console.log("error stack: " + err.stack);
            })
```
- 実行ログ
```
ncmb.min.js:36510 This is not supported in browser version of superagent
Request.agent @ ncmb.min.js:36510
(anonymous) @ ncmb.min.js:420
module.exports @ ncmb.min.js:420
File.download @ ncmb.min.js:394
(anonymous) @ index.js:54
index.js:59 error : Parser is unable to parse the response
index.js:60 error stack: Error: Parser is unable to parse the response
    at Request.<anonymous> (file:///android_asset/www/js/ncmb.min.js:36274:13)
    at Request.Emitter.emit (file:///android_asset/www/js/ncmb.min.js:11462:20)
    at XMLHttpRequest.xhr.onreadystatechange (file:///android_asset/www/js/ncmb.min.js:36606:10)
```

--- a/lib/request.js
+++ b/lib/request.js
@@ -201,7 +201,8 @@ module.exports = function(opts, callback){
           });
         }
         if(responseType && r.responseType) r.responseType(responseType);
-        if (method === "GET" && path.indexOf("/2013-09-01/files/") != -1) {
+        var isNodeEv = new Function("try { return this === global; } catch(e) { return false; }");
+        if (isNodeEv() && method === "GET" && path.indexOf("/2013-09-01/files/") != -1) {
           if (r.buffer) {
             r.buffer(true).parse(function(res, callback) {
               res.data = '';

